core/src/iam/signin.rs

```diff
@@ -21,20 +23,20 @@ pub async fn signin(
 	// Parse the specified variables
 	let ns = vars.get("NS").or_else(|| vars.get("ns"));
 	let db = vars.get("DB").or_else(|| vars.get("db"));
-	let sc = vars.get("SC").or_else(|| vars.get("sc"));
+	let ac = vars.get("AC").or_else(|| vars.get("ac"));
 
 	// Check if the parameters exist
-	match (ns, db, sc) {
-		// SCOPE signin
-		(Some(ns), Some(db), Some(sc)) => {
+	match (ns, db, ac) {
+		// DB signin with access method
+		(Some(ns), Some(db), Some(ac)) => {
 			// Process the provided values
 			let ns = ns.to_raw_string();
 			let db = db.to_raw_string();
-			let sc = sc.to_raw_string();
-			// Attempt to signin to specified scope
-			super::signin::sc(kvs, session, ns, db, sc, vars).await
+			let ac = ac.to_raw_string();
+			// Attempt to signin using specified access method
+			super::signin::db(kvs, session, ns, db, ac, vars).await
```

Comment:
Should `super::signin::db` be more descriptive, something like `super::signin::db_rd` perhaps?